src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Study.java

```diff
@@ -83,6 +83,11 @@
     @Column
     private boolean browsable;
 
+    @ApiModelProperty(position = 9)
+    @JsonProperty
+    @OneToMany
+    private List<Study> linkedStudies;
```

Comment:
Given these should be the children studies only, we should the attribute that way. Otherwise it will sound like the `StudyService.findLinkedStudies` method simply returns the value of this attribute, when it is not the case.

Reply:
OK! I have renamed the attribute to `childStudies` as discussed in 07d0b24